posts/2015-04-10-symfony-live-2015.md

```diff
@@ -0,0 +1,214 @@
+* footprint pour chaque user
+
+Faire attention :
+* à nettoyer le cookie header quand le hash est fetché
+* à vérifier que les Vary headers sont corrects
+
+Combiner plusieurs Vary Header :
+* Différentes règles de cache : no caching, Vary: user context, Vary: Cookie
+* ESI de différentes règles pour des fragments spécifiques
+
+La présentation se termine par un use case sous forme d'une pièce de théâtre assez drôle avec des vraies cookies (oui le biscuit) : accès à un résumé d'article (user anonymous) ou à l'article complet (user authentifié).
+
+Ressources :
+* [Voir la doc de FOSHttpCacheBundle](https://github.com/FriendsOfSymfony/FOSHttpCacheBundle)
+* et plus spécifiquement [la doc sur le User Context](https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/Resources/doc/features/user-context.rst)
+
+
+## Laisse pas traîner ton log ! par Olivier Dolbeau
+
+Introduction à comment tirer parti des logs grâce à Monolog et à la stack ELK (Elasticsearch / Logstash / Kibana) et en faire des dashboards.
+
+En savoir plus : [Speakerdeck Laisse pas traîner ton log !](https://speakerdeck.com/odolbeau/laisse-pas-trainer-ton-log)
+
+
+## Retour d’expérience : attention chérie, ça va trancher par Bastien Jaillot
+
+Pitch :
+
+> Vous vous sentez comme un membre d’équipage dont le navire est sur le point de se briser ? Votre environnement de travail est si triste que vous n’avez plus confiance en l'humanité ? Google vous propose une solution : http://goo.gl/I4jM4n Notre histoire, se déroule au sein d’une équipe technique au bord de l'apoplexie tant en terme humain que technique. Elle va progressivement remonter la pente et gagner ses lettres de noblesses pour enfin brandir son glaive et s’écrier “For The Victory!”. Comment passer d’un monolithe à une architecture microservices (à base de composants Symfony2) ? En quoi l’arrivée d’un chef de projet et de nouveaux process nous a sauvé la mise ? En quoi l’attention porté sur l'humain a favorisé l’émergence d’un collectif technique ? Comment favoriser l’insertion d’une nouvelle personne dans l’équipe ? En quoi l'utilisation pragmatique de technologies reconnues et émergeantes (elk, docker) nous a aidé au jour le jour.
+
+Retour d'expérience sur la refonte architecture de MediaPart d'une archi monolithique Drupal6 vers des microservices basés sur ElasticSearch et Symfony2.
+"Tout logiciel reflète l'organisation qui l'a créée" (loi de Conway). Tout ce qui se rapporte à l'informatique arrive aux développeurs. Le SI est devenu lui même un monolithe.
+
+Il s'agit non pas d'une présentation technique mais un retour d'expérience sur le changement.
+
+Electro-choc et solution pour amorcer le changement :
+* démarrer un projet ambitieux
+* une personne pour gérer le "bruit" (gestion du parc SI)
+* ne plus corriger les anomalies non bloquantes, ni d'évolutions. Site en jachère.
+* redémarrer le développement du site en partant sur de meilleures bases et recrutement de profils spécialisés développement web.
```

Comment:
The list lines above start with lowercase where some others begin with capital letters. I don't know what is the rule but I think this could be unified.